webapp/src/components/App/Singlestudy/explore/Configuration/AdvancedParameters/utils.ts

```diff
@@ -0,0 +1,184 @@
+// Constants
+////////////////////////////////////////////////////////////////
+
+export const SPATIAL_CORRELATIONS_OPTIONS = Object.values(SpatialCorrelations);
+export const INITIAL_RESERVOIR_OPTIONS = Object.values(InitialReservoirLevels);
+export const HYDRO_HEURISTIC_POLICY_OPTIONS =
+  Object.values(HydroHeuristicPolicy);
+export const HYDRO_PRICING_MODE_OPTIONS = Object.values(HydroPricingMode);
+export const POWER_FLUCTUATIONS_OPTIONS = Object.values(PowerFluctuations);
+export const SHEDDING_POLICY_OPTIONS = Object.values(SheddingPolicy);
+export const RESERVE_MANAGEMENT_OPTIONS = Object.values(ReserveManagement);
+export const UNIT_COMMITMENT_MODE_OPTIONS = Object.values(UnitCommitmentMode);
+export const SIMULATION_CORES_OPTIONS = Object.values(SimulationCores);
+export const RENEWABLE_GENERATION_OPTIONS = Object.values(
+  RenewableGenerationModeling
+);
+
+////////////////////////////////////////////////////////////////
+// Types
+////////////////////////////////////////////////////////////////
+
+export interface IAdvancedParameters {
+  accuracyOnCorrelation: string;
+}
+
+export interface IOtherPreferences {
+  initialReservoirLevels: string;
+  hydroHeuristicPolicy: string;
+  hydroPricingMode: string;
+  powerFluctuations: string;
+  sheddingStrategy: string;
+  sheddingPolicy: string;
+  unitCommitmentMode: string;
+  numberOfCoresMode: string;
+  renewableGenerationModelling: string;
+  dayAheadReserveManagement: string;
+}
+
+export interface IGeneralData {
+  otherPreferences: IOtherPreferences;
+  advancedParameters: IAdvancedParameters;
+  seedsMersenneTwister: { [key: string]: number };
+}
+
+export interface AdvancedParamsFormFields {
+  accuracyOnCorrelation: string[];
+  adequacyBlockSize: number;
+  dayAheadReserveManagement: string;
+  hydroHeuristicPolicy: string;
+  hydroPricingMode: string;
+  initialReservoirLevels: string;
+  numberOfCoresMode: string;
+  powerFluctuations: string;
+  renewableGenerationModelling: string;
+  seedHydroCosts: number;
+  seedInitialReservoirLevels: number;
+  seedSpilledEnergyCosts: number;
+  seedThermalCosts: number;
+  seedTsgenHydro: number;
+  seedTsgenLoad: number;
+  seedTsgenSolar: number;
+  seedTsgenThermal: number;
+  seedTsgenWind: number;
+  seedTsnumbers: number;
+  seedUnsuppliedEnergyCosts: number;
+  sheddingPolicy: string;
+  sheddingStrategy: string;
+  unitCommitmentMode: string;
+}
+
+////////////////////////////////////////////////////////////////
+// Functions
+////////////////////////////////////////////////////////////////
+
+/**
+ * Translate all top-level keys of the given `object` to camelCase.
+ *
+ * @param {Object} object
+ *
+ * @returns {Object}
+ */
+
+function keysToCamelCase<T>(object: Record<any, any>): T {
+  return Object.keys(object).reduce((acc, key) => {
+    if (RA.isString(key)) {
+      acc[_.camelCase(key)] = object[key];
+    }
+    return acc;
+  }, {} as Record<any, any>) as T;
```

Comment:
put this in services/utils/index.ts (and add linter exception to prevent warning for the `any` type)